capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CaptalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] You are expected to Capitalize your name, not make it UPPERCASE. Kindly refactor your method to return 'Maximilianus', not 'MAXIMILIANUS' as we have in the image below.

  ![rizo-2](https://user-images.githubusercontent.com/36542986/164206193-31ffeeb2-4e18-42ff-8b00-c5ebb07a9f3b.jpg)